samcli/local/apigw/event_constructor.py

```diff
@@ -0,0 +1,327 @@
+"""
+Lambda event construction and generation
+"""
+
+import base64
+import json
+import logging
+from datetime import datetime
+from time import time
+
+from samcli.local.apigw.path_converter import PathConverter
+from samcli.local.events.api_event import (
+    ApiGatewayLambdaEvent,
+    ApiGatewayV2LambdaEvent,
+    ContextHTTP,
+    ContextIdentity,
+    RequestContext,
+    RequestContextV2,
+)
+
+LOG = logging.getLogger(__name__)
+
+
+class EventConstructor:
```

Comment:
question for thought, but not necessarily requiring any changes:
if all of the methods of this class are static methods, does this need to be a class (vs. just a module with functions)?

Reply:
Initially stuffed it into a class to reduce the amount of importing, but looking at it again we can just make these standalone functions since only two of these methods are exposed in other parts of the CLI